JAVASCRIPT/ASSIGNMENT_1_1_2_22/index.html

```diff
@@ -0,0 +1,119 @@
+</head>
+<body>
+
+  <style>
+    .row{
+      margin: 20px;
+    }
+    .row>.col-12{
+      display: flex;
+      justify-content: center;
+    }
+    button{
+      margin: 10px;
+    }
+    .card .card-body{
+      width: 700px !important;
+    }
+  </style>
+
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="container">
+      <a class="navbar-brand" href="#">Loan system</a>
+      <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbar-menu">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link "href="#"></a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <div class="container">
+    <div class="row">
+      <div class="col-12">
+        <div class="card">
+          <div class="card-body">
+            <h4 class="card-title">Emi Calculator</h4>
+            <form action="#" name="loan_form" onsubmit=" return calculate()">
+              <div class="form-group">
+                <label>Applicant Name</label>
+                <input type="text" class="form-control" name="name">
+              </div>
+              <div class="form-group">
+                <label>Loan Amount</label>
+                <input type="number" class="form-control" name="amount">
+              </div>
+              <div class="form-group">
+                <label>Loan type</label>
+                <select class="form-control" name="type" onchange="setRateAndDuration(this.value)">
+                  <option selected>Select Type</option>
+                  <option value="1">Home</option>
+                  <option value="2">Car</option>
+                  <option value="3">Personal</option>
+                </select>
+              </div>
+              <div class="form-group">
+                <label>Duration</label>
+                <input type="number" class="form-control" name="duration" min=1>
+              </div>
+              <div class="form-group">
+                <label>Rate of Intereste</label>
+                <input type="number" class="form-control" name="roi" readonly>
+              </div>
+              <div class="form-group">
+                <label>Emi</label>
+                <input type="text" class="form-control" name="emi">
+              </div>
+              <button  class="btn btn-primary" >Calculate</button>
+              <button class="btn btn-danger" type="reset">Reset</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script>
+    function calculate(){
+      var amount = document.loan_form.amount.value;
```

Comment:
create a variable for  document.loan_form object . Parse from it so you can reduce one traverse tree in DOM operation